hera_sim/io.py

```diff
@@ -29,10 +28,40 @@ def empty_uvdata(
     conjugation=None,
     **kwargs,
 ):
-    # TODO: docstring
-    """"""
+    """Create an empty UVData object with given specifications.
+
+    Parameters
+    ----------
+    Ntimes : int, optional
+        NUmber of unique times in the data object.
+    start_time : float, optional
+        Starting time (Julian date) by default 2456658.5
+    array_layout : [type], optional
+        Specify an array layout.
+    Nfreqs : int, optional
+        Number of frequency channels in the data object
+    start_freq : float, optional
+        Lowest frequency channel, by default None
+    channel_width : float, optional
+        Channel width, by default None
+    n_freq : [type], optional
+        Alias for ``Nfreqs``
+    n_times : [type], optional
+        Alias for ``Ntimes``.
+    antennas : [type], optional
+        Alias for array_layout?
```

Comment:
This is indeed an alias for `array_layout` and is kept purely in the interest of backwards compatibility.

Reply:
Thanks, added that in.